install/install.go

```diff
@@ -283,7 +330,35 @@ func K8sInstaller(c *k8s.Client, o Options) error {
 		if err != nil {
 			return err
 		}
-		fmt.Printf("KubeArmor manifest file saved to \033[1m%s\033[0m\n", f.Name())
+		s3:=f.Name()
+		progbar.filled=48
+		progbar.unfilled=0
+		progbar.done=(float64(progbar.filled)/progbar.size)*float64(100)
+		s:=""
+		for i:=0;i<progbar.filled;i++{
+			s=s+progbar.fill
+		}
+		progbar.bar=s
+		printMessage("🤩  KubeArmor manifest file saved to \033[1m"+s3+"\033[0m",false)
+
+	}
+	fmt.Printf("Checking if Karmor containers are running ...")
+	rcursor:=[4]string{"−","\\","|","/"}
+	rcursorcount:=0
+	for {
+		time.Sleep(300*time.Millisecond)
+		pods, _ := c.K8sClientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{LabelSelector: "kubearmor-app",FieldSelector: "status.phase!=Running"})
+		podno:=len(pods.Items)
+		fmt.Printf("\rKARMOR pods left to run : %d ... %s               ",podno,rcursor[rcursorcount])
```

Comment:
fmt.Printf("\r KubeArmor pods left to run: %d ... %s", podno,rcursor[rcursorcount])

Reply:
I think the trailing spaces are needed to wipe out previous text since carriage return (\r) is used. I see why those are added.

Reply:
Yes , else there would be residues of previous print.

Reply:
Ah cool, we can keep them, please change the display message only.